src/common/res/features/export-transactions/main.js

```diff
@@ -170,34 +186,39 @@
           return !yTrans.get('isTombstone');
         });
 
-        var cleanedTransArray = ynabTransArray.map(cleanTransaction);
-        return cleanedTransArray;
-      }
-
-
-      // Sort an Array of Objects by key
-      function sortByKey(transactions, key, order) {
```

Comment:
Regarding [this comment](https://github.com/blargity/toolkit-for-ynab/pull/199#discussion_r52838123), I meant this bug already existed before, and fixing the bug will change the behavior of the Export to CSV feature even if no split transactions were present. I'll fix it now, and let me know if that will be a problem.